aten/src/ATen/cpu/vec256/vec256_complex_double_no_vec.h

```diff
@@ -0,0 +1,328 @@
+#pragma once
+
+#include <ATen/cpu/vec256/intrinsics.h>
+#include <ATen/cpu/vec256/vec256_base.h>
+#if defined(__AVX__) && !defined(_MSC_VER)
+#include <sleef.h>
+#endif
+
+namespace at {
+namespace vec256 {
+// See Note [Acceptable use of anonymous namespace in header]
+namespace {
+
+template <> class Vec256<std::complex<double>> {
+private:
+  std::complex<double> values[2] = {0};
+public:
+  using value_type = std::complex<double>;
+  static constexpr int size() {
+    return 2;
```

Comment:
I'm not sure what the correct number here should be, but the fact that you've written 2 here and below is a huge smell. In the base implementation, the size is `32 / sizeof(T)`. `sizeof(std::complex<double>)` is 16, so I expect 2 here (good!) but 4 in the definition for `std::complex<float>`